packages/service-clients/end-to-end-tests/azure-client/package.json

```diff
@@ -72,19 +72,14 @@
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/counter": "workspace:~",
 		"@fluidframework/datastore-definitions": "workspace:~",
-		"@fluidframework/fluid-static": "workspace:~",
-		"@fluidframework/map": "workspace:~",
-		"@fluidframework/map-legacy": "npm:@fluidframework/map@^1.4.0",
-		"@fluidframework/matrix": "workspace:~",
 		"@fluidframework/runtime-definitions": "workspace:~",
-		"@fluidframework/sequence": "workspace:~",
 		"@fluidframework/telemetry-utils": "workspace:~",
 		"@fluidframework/test-runtime-utils": "workspace:~",
 		"@fluidframework/test-utils": "workspace:~",
-		"@fluidframework/tree": "workspace:~",
 		"axios": "^1.7.7",
 		"cross-env": "^7.0.3",
-		"fluid-framework": "^1.4.0",
+		"fluid-framework": "workspace:~",
+		"fluid-framework-legacy": "npm:fluid-framework@^1.4.0",
```

Comment:
Note: merging this would have the nice side-effect of _almost_ fixing the issues with the service client test pipeline even before we address the problem in `flub` because `"fluid-framework": "workspace:~"` would be replaced with the correct version, and `"fluid-framework-legacy"` would not be replaced, keeping the reference to 1.4.0. However, given our current feed setup in ADO that would still result in a failure to download `fluid-framework@2.x.y-<buildId>`. I'm addressing that separately.